tests/cli/workflow/raredisease/test_cli_raredisease_compound_commands.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from _pytest.logging import LogCaptureFixture
+from click.testing import CliRunner
+from cg.cli.workflow.raredisease.base import (
+    raredisease,
+)
+from cg.constants import EXIT_SUCCESS
+from cg.models.cg_config import CGConfig
+
+
+def test_raredisease_no_args(cli_runner: CliRunner, raredisease_context: CGConfig):
```

Comment:
there are 3 tests that are basically the same for rnafusion, taxprofiler and raredisease. I would refactor so this becomes a single parametrised test. 

I'm thinking something like:
- add a new file: cg/tests/cli/workflow/nf_analysis/test_cli.py
- make a parametrised function similar to:

```
@pytest.mark.parametrize(
    "pipeline, context",
    [(Pipeline.RNAFUSION, "rnafusion_context"),
     (Pipeline.TAXPROFILER, "taxprofiler_context"),
     (Pipeline.RAREDISEASE, "raredisease_context"),
    ],
)
def test_workflow_no_args(cli_runner: CliRunner, pipeline: Pipeline, context: CGConfig):  
    """Test to assess that NF workflows without options successfully print help and exist without error."""
    # GIVEN no arguments or options besides the command call

    # WHEN running command
    result = cli_runner.invoke(pipeline, [], obj=context)

    # THEN command runs successfully
    print(result.output)

    assert result.exit_code == EXIT_SUCCESS

    # THEN help should be printed
    assert "help" in result.output

```

Reply:
That sounds smart, I will make it a separate PR, though 👍